locations/spiders/digi_telecommunications_my.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+from locations.hours import DAYS_FULL, OpeningHours
+from locations.storefinders.storepoint import StorepointSpider
+
+
+class DiGiTelecommunicationsMYSpider(StorepointSpider):
+    name = "digi_telecommunications_my"
+    item_attributes = {"brand": "DiGi Telecommunications", "brand_wikidata": "Q3268530"}
+    key = "1624a4f7eafca0"
+
+    def parse_item(self, item, location: {}, **kwargs):
+        item["addr_full"] = item.pop("street_address")
```

Comment:
If this is common across all StorepointSpider spiders, it should go in `StorepointSpider`. Same goes for the opening hours.

Reply:
It does indeed look common with street_address and addr_full, as well as the opening hours format. I'll change the Storepoint spider.